thirdparty/poppler/qt5/poppler-optcontent.cc

```diff
@@ -427,4 +427,4 @@ namespace Poppler
   }
 }
 
-#include "poppler-optcontent.moc"
+//#include "poppler-optcontent.moc"
```

Comment:
It does (or rather should) get build, it is not a file that exists in the repository.